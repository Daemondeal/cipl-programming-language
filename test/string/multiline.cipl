let a = "1
2
3
4"

print("Start Multiline")
print(a)
print("End Multiline")