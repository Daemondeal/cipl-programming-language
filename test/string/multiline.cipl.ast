(let a 1
2
3
4)
(expression (call print Start Multiline))
(expression (call print a))
(expression (call print End Multiline))

