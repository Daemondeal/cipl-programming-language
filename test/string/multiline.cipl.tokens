Token          Lexeme         Line 
LET            let            1    
IDENTIFIER     a              1    
EQUAL          =              1    
STRING         "1\n2\n3\n4"   4     1
2
3
4
LINE_END       \n             5    
IDENTIFIER     print          6    
LEFT_PAREN     (              6    
STRING         "Start Multiline"6     Start Multiline
RIGHT_PAREN    )              6    
LINE_END       \n             7    
IDENTIFIER     print          7    
LEFT_PAREN     (              7    
IDENTIFIER     a              7    
RIGHT_PAREN    )              7    
LINE_END       \n             8    
IDENTIFIER     print          8    
LEFT_PAREN     (              8    
STRING         "End Multiline"8     End Multiline
RIGHT_PAREN    )              8    
LINE_END       \n             8    
EOF                           8    
