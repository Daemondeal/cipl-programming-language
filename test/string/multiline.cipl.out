::stdout::
Start Multiline
1
2
3
4
End Multiline

::stderr::
